tests/kafkatest/services/kafka/kafka.py

```diff
@@ -185,7 +185,11 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
 
         for node in self.nodes:
             node.version = version
-            node.config = KafkaConfig(**{config_property.BROKER_ID: self.idx(node)})
+            node.config = KafkaConfig(**{
+                config_property.BROKER_ID: self.idx(node),
+                config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: zk_connect_timeout,
```

Comment:
Could we change the default ZK session timeout and ZK connection timeout in line 101 to 18 secs too?